app/views/facility_orders/show.html.haml

```diff
@@ -10,11 +10,11 @@
     = banner_label @order, :created_by_user
     - if SettingsHelper.feature_on?(:cross_core_projects)
       - project_id = @cross_core_order_details_by_facility.present? ? @order_details.first&.project_id : nil
-      - if project_id
+      - if @cross_core_project_name
```

Comment:
based on spec failures I think maybe we need:
```suggestion
      - if @cross_core_project_name && project_id
```
... but not sure if there is a valid case where we have a cross core project with no order details